app/pulp/app/serializers/repository.py

```diff
@@ -46,11 +46,72 @@ class ImporterSerializer(MasterModelSerializer):
 
     Every subclass must override the `_href` field with a `RepositoryNestedIdentityField` that
     defines the view_name.
+
+    `repository` is defined on the `Importer` model, but is excluded here because the user must
+    already know about a `Repository` and its relationship to the Importer because the REST
+    endpoint includes `Repository.name`.
```

Comment:
I'm not sure that this belongs in the docstring at all, really. I think a comment where the repository field would normally go (probably right down around _href) is enough to let devs know that it was omitted on purpose, but it's not a necessary part of the ImporterSerializer interface as declared here so much as a non-obvious implementation detail. Similar to @goosemania, though, maybe it's just me.

Something like `# The repository related field is excluded from this serializer because all of its API views exist in the context of its related repository` might be fine.